### PR TITLE
citus_tables and citus_shards

### PR DESCRIPTION
**Pull request by jonels-msft**

### CODE REVIEW

File: develop/api_metadata.rst [hunk 1]
```diff
@@ -110,6 +110,40 @@ The shardstorage column in pg_dist_shard indicates the type of storage used for
...
+.. code-block:: sql
+
+  SELECT * FROM citus_shards;
+
+::
+
+  .
+   shardid | table_name | citus_table_type | colocation_id | nodename  | nodeport | shard_size | shard_size_bytes
+  ---------+------------+------------------+---------------+-----------+----------+------------+------------------
+    102170 | dist       | distributed      |            34 | localhost |     9701 | 86 MB      |         90677248
```
Comment (SaitTalhaNisanci) [inline develop/api_metadata.rst:135]: the output is a bit different now. see https://github.com/citusdata/citus/pull/4649.

Basically the shard_size is now in byte format that way it is easy to do operations on it (a bit hard when it is prettified).

Also the order is a bit different now

**Comment by jonels-msft** [comment]
Thanks @SaitTalhaNisanci, can you give this another review?

File: admin_guide/diagnostic_queries.rst [hunk 2]
```diff
@@ -161,28 +158,21 @@ This query gets a list of the sizes for each distributed table plus the size of
...
-  SELECT
-    tablename,
-    pg_size_pretty(
-      citus_total_relation_size(tablename::text)
-    ) AS total_size
-  FROM pg_tables pt
-  JOIN pg_dist_partition pp
-    ON pt.tablename = pp.logicalrelid::text
-  WHERE schemaname = 'public';
+  SELECT Name, Size
```
Comment (SaitTalhaNisanci) [inline admin_guide/diagnostic_queries.rst:161]: These have also changed I believe `table_name`, `table_size`.

File: develop/api_metadata.rst [hunk 3]
```diff
@@ -110,6 +110,40 @@ The shardstorage column in pg_dist_shard indicates the type of storage used for
...
 +----------------+----------------------+-----------------------------------------------------------------------------+
 
+.. _citus_shards:
+
+Shard information view
+~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to the low-level shard metadata table described above, Citus provides a ``citus_shards`` view to easily check:
+
+* Where each shard is (node, and port),
```
Comment (SaitTalhaNisanci) [inline develop/api_metadata.rst:120]: there is also the colocation id in the output, not sure if it is worth adding here.

Comment (SaitTalhaNisanci) [reply develop/api_metadata.rst:120]: Ah you already explained it below :)

File: develop/api_metadata.rst [hunk 4]
```diff
@@ -327,6 +361,37 @@ Here's an example of how ``create_distributed_function()`` adds entries to the
...
+Here's an example:
+
+.. code-block:: sql
+
+  SELECT * FROM citus_tables;
+
+::
+
+  ┌──────────┬──────────────────┬─────────────────────┬───────────────┬─────────┬─────────────┬───────┬───────────────┐
+  │   Name   │ Citus Table Type │ Distribution Column │ Colocation ID │  Size   │ Shard Count │ Owner │ Access Method │
```
Comment (SaitTalhaNisanci) [inline develop/api_metadata.rst:388]: The names will also need to be updated.

**Review by SaitTalhaNisanci — APPROVED** [review]

File: develop/api_metadata.rst (hunk 4 above)
Comment (jonels-msft) [reply develop/api_metadata.rst:388]: Oops, I was looking at an old version of the code, before Marco's PR.

Comment (SaitTalhaNisanci) [reply develop/api_metadata.rst]: Where in `src/backend/distributed/build/sql/citus--9.5-1--10.0-1.sql` does it have the old names?

Comment (jonels-msft) [reply develop/api_metadata.rst]: Sorry I think I was mistaken. Not sure what I had open in my editor, must have been src/backend/distributed/sql/udfs/citus_tables/10.0-1.sql prior to commit 
https://github.com/citusdata/citus/commit/03328e9679e08685f8e24f77a577b17a4b9176b9

**Pull request merged**